README.md

```diff
@@ -212,7 +212,7 @@ Now copy-paste the following *sample code* to get started:
 #### Open index.html in your Browser
 
 When you ***open*** `index.html` in your ***web browser***
-you should expect to see something like this: (_without the annotations_)
+you should expect to see something like this: (_without the green annotations_)
```

Comment:
The annotations here are both green (on passing tests) and red (on failing tests) so we'll need to change this sentence either back to what it was or to be more explicit on the green and red annotations (not to be confused with QUnit's own green and red defaults) so as not to be misleading ☺️